CHANGES.md

```diff
@@ -35,6 +35,10 @@
   * Fixed dunst notifications being obscured when moving floats.
     https://github.com/xmonad/xmonad/issues/208
 
+  * `util/GenerateManpage.hs` is no longer distributed in the tarball.
+    Instead, the manpage source is renegerated and manpage rebuilt
```

Comment:
```suggestion
    Instead, the manpage source is regenerated and manpage rebuilt
```

Reply:
Oh, stupid vim treats indented bullets as code blocks and refuses to spellcheck in there. :-(